### ISSUE TITLE
[5.x] Fatal Error cli

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#40770]
### Steps to reproduce the issue
try to run the cli eg `php cli/joomla.php`

### Actual result
```
PHP Fatal error:  Declaration of Joomla\Console\Descriptor\TextDescriptor::describe(Symfony\Component\Console\Output\OutputInterface $output, $object, array $options = []) must be compatible with Symfony\Component\Console\Descriptor\Descriptor::describe(Symfony\Component\Console\Output\OutputInterface $output, object $object, array $options = []): void in C:\laragon\www\j5\libraries\vendor\joomla\console\src\Descriptor\TextDescriptor.php on line 37
Symfony\Component\ErrorHandler\Error\FatalError^ {#693
  #message: "Compile Error: Declaration of Joomla\Console\Descriptor\TextDescriptor::describe(Symfony\Component\Console\Output\OutputInterface $output, $object, array $options = []) must be compatible with Symfony\Component\Console\Descriptor\Descriptor::describe(Symfony\Component\Console\Output\OutputInterface $output, object $object, array $options = []): void"
  #code: 0
  #file: "C:\laragon\www\j5\libraries\vendor\joomla\console\src\Descriptor\TextDescriptor.php"
  #line: 37
  -error: array:4 [
    "type" => 64
    "message" => "Declaration of Joomla\Console\Descriptor\TextDescriptor::describe(Symfony\Component\Console\Output\OutputInterface $output, $object, array $options = []) must be compatible with Symfony\Component\Console\Descriptor\Descriptor::describe(Symfony\Component\Console\Output\OutputInterface $output, object $object, array $options = []): void"
    "file" => "C:\laragon\www\j5\libraries\vendor\joomla\console\src\Descriptor\TextDescriptor.php"
    "line" => 37
  ]
}
```

![image](https://github.com/joomla/joomla-cms/assets/1296369/5be25dfe-564c-448c-9cf9-1fd1d1dfbc9f)

### System information (as much as possible)
PHP 8.1.3 (cli) (built: Feb 16 2022 08:20:53) (NTS Visual C++ 2019 x64)

**Comment by HLeithner** [comment]
have to be fixed in the framework https://github.com/joomla-framework/console/tree/3.x-dev @Hackwar 

I leave this open till it's fixed in the framework and here in the cms

**Comment by Hackwar** [comment]
The issue is fixed in the framework. You have to update the dependencies to take effect in J5

**Comment by richard67** [comment]
Should be solved with PR #41028 , which includes the dependency update. Leaving this issue open though as long as it’s not clear that the dependency update will not be reverted.

**Comment by richard67** [comment]
Has been solved with the dependency update from PR #41028 .
